### ISSUE TITLE
Slow android tab changes

### ISSUE THREAD

**Issue by michaltaberski** [react-native-elements/react-native-elements-app#27]
Hey,

I am playing with the app on iPhone 7 and Nexus 5X. On both devices I have installed the app via Expo. 

On iPhone the app is working decently, but on Nexus its quite slow. I wonder what's the reason. 

- is it navigation lib slow?
- is it because of the expo?
- react-native-elements are too heavy?

I am looking for a right tech stack for the app I plan to build and I am confused with the performance issues of the demo app.

Video clip where I recorded how slow the app on Nexus is: http://take.ms/gi6ET

**Comment by dyaacov** [comment]
Happens to me as well, I'm using Nexus 5x
Not only tabs, also button1 button2 in Components page

**Comment by Monte9** [comment]
@michaltaberski not sure why this would be an issue.

Just fyi, this library has been moved into React Native Elements core library under `/example` folder. 
We have reimplemented the entire app from scratch using Expo & `react-navigation` and also updated it to include all the latest component for RNE. 

- Here's the link to the app: https://expo.io/@monte9/react-native-elements-app
- Also here's the new home for this repo: https://github.com/react-native-training/react-native-elements/tree/master/example
